### ISSUE TITLE
No fx

### ISSUE THREAD

**Issue by kcmojack** [byxor/thug-pro-reshade#2]
no matter what i do there isnt any fx. ive tried reinstalling multiple times with no avail.
![image](https://github.com/byxor/thug-pro-reshade/assets/102624004/1e24cc64-6ea1-4029-95bd-813b5271f6b0)

**Comment by byxor** [comment]
I wonder if you have other conflicting files in your THUG Pro directory. e.g. files from an older version of Reshade.

If you get the time, could you run the`dir` command in your THUG Pro folder and paste the output here? Just want to see if you have extra unwanted files.

You can open Powershell quickly from explorer with `File` -> `Open Windows Powershell`:
<img width="688" alt="image" src="https://github.com/byxor/thug-pro-reshade/assets/14977217/3531425a-091a-4022-aa9f-aedbdaf39245">

**Comment by byxor** [comment]
Multiple people are encountering this issue now. Not sure if it's a local setup issue or if it's broken now for a lot of people. I don't use it anymore so I haven't noticed any issues. Don't see why mine would stop working unless I updated Windows or DirectX in some way

**Comment by odiplol** [comment]
Yeah with this issue for me, i run windows 10 ltsc, and when i tried installing thug pro it wouldnt recognise my directx, i installed a different version with tons of tools and stuff and then it finally worked, maybe it has something to do with this version of directx.

**Comment by odiplol** [comment]
Guys, I found a SOLUTION!!! if you go to settings in the reshade menu, and select add effect directory path, select the effects folder in "reshade-shaders" and then press refresh in the reshade home menu.

**Comment by byxor** [comment]
Only seeing this now, nice catch. I should probably mention this in the README, unless there's a way I can fix it directly
